astropy/wcs/utils.py

```diff
@@ -118,6 +143,76 @@ def _wcs_to_celestial_frame_builtin(wcs):
                 representation_type=SphericalRepresentation,
                 obstime=wcs.wcs.dateobs or None,
             )
+        elif (
+            "LN" in xcoord
+            and "LT" in ycoord
+            and "H" not in xcoord
+            and "CR" not in xcoord
+        ):
+            # Coordinates on a planetary body, as defined in
+            # https://agupubs.onlinelibrary.wiley.com/doi/10.1029/2018EA000388
+
+            object_name = SOLAR_SYSTEM_OBJ_DICT.get(xcoord[:2])
+            if object_name is None:
+                raise KeyError(f"unknown solar system object abbreviation {xcoord[:2]}")
+
+            if WCSLIB_LT8:
+                warnings.warn(
+                    f" {_wcs.__version__} does not support planetary WCS,"
```

Comment:
If we need 8.1, then that should be the version check too, i.e., `WCSLIB_LT_8_1`